homeassistant/components/nuki/__init__.py

```diff
@@ -110,6 +110,7 @@ async def handle_webhook(
             translation_placeholders={
                 "base_url": hass_url,
                 "network_link": "https://my.home-assistant.io/redirect/network/",
+                "sample_ip": "http://192.168.1.10:8123",
```

Comment:
This is not an IP. It's a URL.
It should instead be called `sample_url`.

You will need to provide both a sample IP and a sample URL for the issue.